ggml-rpc.cpp

```diff
@@ -71,6 +72,7 @@ struct rpc_tensor {
     uint64_t data;
     char name[GGML_MAX_NAME];
 };
+#pragma pack(pop)
```

Comment:
Aren't these pragmas valid only for MSVC? I think for GCC you might want to use `__attribute__((__packed__))`

Reply:
yeah, valid in gcc, not in std tho, https://gcc.gnu.org/onlinedocs/gcc-4.4.4/gcc/Structure_002dPacking-Pragmas.html